feat: [IA-406] Fixed iOS accessibility internal translations

dgopsq opened the pull request:
## Short description
This PR is going to patch the React Native NPM package, adding an Italian localisation file to translate various internal strings used for accessibility.

The patch will just **add** a new file inside the React Native module, not changing anything else.

For reference see [this blog post](https://valtterilaine.medium.com/react-native-accessibilityrole-i18n-support-for-ios-and-android-16d896e69178).

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/5963683/155965869-80ab3249-103d-49fa-9f1d-41e37b216cbf.png">

## List of changes proposed in this pull request
- Translated the `checked` and `not checked` values in: `selezionato` and `non selezionato`

## How to test
Using the accessibility inspector to analyse the services configuration's radio buttons while using iOS in Italian should result in having the `checked` and `not checked` values translated as `selezionato` and `non selezionato`.

dgopsq commented:
@thisisjp I added you as a reviewer too because there's translated text but not in the usual place (https://github.com/pagopa/io-app/pull/3791/files#diff-5f8f59afee259f80d174c30a791f101d87b9bc1086431df7a82e41345bde40a2R27-R28)

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"tab"="tab description";
++"tablist"="tab list";
++"timer"="timer";
++"toolbar"="tool bar";
```
```suggestion
+"toolbar"="barra degli strumenti";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"toolbar"="tool bar";
++"checked"="selezionato";
++"unchecked"="non selezionato";
++"busy"="busy";
```
```suggestion
+"busy"="occupato";
```

thisisjp replied:
```suggestion
+"busy"="occupato";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"busy"="busy";
++"expanded"="expanded";
++"collapsed"="collapsed";
++"mixed"="mixed";
```
```suggestion
+"mixed"="misto";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"checked"="selezionato";
++"unchecked"="non selezionato";
++"busy"="busy";
++"expanded"="expanded";
```
```suggestion
+"expanded"="espanso";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"unchecked"="non selezionato";
++"busy"="busy";
++"expanded"="expanded";
++"collapsed"="collapsed";
```
```suggestion
+"collapsed"="compresso";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++  React
++*/
++"alert"="alert";
++"checkbox"="checkbox";
```
```suggestion
+"checkbox"="casella di controllo";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"menubar"="menu bar";
++"menuitem"="menu item";
++"progressbar"="progress bar";
++"radio"="radio button";
```
```suggestion
+"radio"="pulsante di scelta";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"menu"="menu";
++"menubar"="menu bar";
++"menuitem"="menu item";
++"progressbar"="progress bar";
```
```suggestion
+"progressbar"="barra di avanzamento";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"progressbar"="progress bar";
++"radio"="radio button";
++"radiogroup"="radio group";
++"scrollbar"="scroll bar";
```
```suggestion
+"scrollbar"="barra di scorrimento";
```

thisisjp commented on patches/react-native+0.64.2.patch:
```diff
@@ -1,14 +1,46 @@
++"combobox"="combo box";
++"menu"="menu";
++"menubar"="menu bar";
++"menuitem"="menu item";
```
```suggestion
+"menuitem"="voce del menu";
```

thisisjp reviewed (commented):
@dgopsq I noticed there are other UI elements which required translation... I didn't go through all of them, but better than nothing, right? :)

fabriziofff reviewed (changes requested):
Could you update the https://github.com/pagopa/io-app/blob/master/patches/patches.md#react-native0642 file, in order to track the reason of this patch?

dgopsq commented:
Thank you @thisisjp, I integrated everything 🙂 Also changed the PR title to a more generic one since we're going to address more translations than the ones in the ticket!

@fabriziofff I just added a small changelog, with a reference to this issue!
